.pre-commit-config.yaml

```diff
@@ -1,8 +1,9 @@
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.3.0
+    rev: v4.4.0
     hooks:
     -   id: check-yaml
+    -   id: no-commit-to-branch
```

Comment:
For details see the documentation of the [no-commit-to-branch hook](https://github.com/pre-commit/pre-commit-hooks#no-commit-to-branch):

> Both main and master are protected by default if no branch argument is set.

So main should be protected by default. No need to pass extra arguments.